script/bench/junit-diff.py

```
@@ -0,0 +1,80 @@
+"""
+
+import sys
+import xml.etree.ElementTree as ET
+from collections import defaultdict
+
+
+def load(path):
+    """Return {(suite, test): seconds} and {suite: seconds} from a JUnit file."""
+    tests = {}
+    suites = defaultdict(float)
+    root = ET.parse(path).getroot()
```

Ruff (0.15.21)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)